<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWnd</class>
 <widget class="QMainWindow" name="MainWnd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>743</width>
    <height>504</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::ContextMenuPolicy::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>MainWnd</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="focusPolicy">
       <enum>Qt::FocusPolicy::ClickFocus</enum>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="contextMenuPolicy">
    <enum>Qt::ContextMenuPolicy::NoContextMenu</enum>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="contextMenuPolicy">
    <enum>Qt::ContextMenuPolicy::NoContextMenu</enum>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="rectangle_create_action"/>
   <addaction name="triangle_create_atcion"/>
   <addaction name="ellipse_create_action"/>
   <addaction name="separator"/>
   <addaction name="relation_create_action"/>
   <addaction name="separator"/>
   <addaction name="move_action"/>
   <addaction name="delete_action"/>
   <addaction name="separator"/>
   <addaction name="load_action"/>
   <addaction name="save_action"/>
  </widget>
  <action name="load_action">
   <property name="text">
    <string>Загрузить</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="save_action">
   <property name="text">
    <string>Сохранить</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <actiongroup name="GroupTool">
   <action name="rectangle_create_action">
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>Прямоугольник</string>
    </property>
    <property name="menuRole">
     <enum>QAction::MenuRole::NoRole</enum>
    </property>
   </action>
   <action name="triangle_create_atcion">
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>Треугольник</string>
    </property>
    <property name="menuRole">
     <enum>QAction::MenuRole::NoRole</enum>
    </property>
   </action>
   <action name="ellipse_create_action">
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>Эллипс</string>
    </property>
    <property name="menuRole">
     <enum>QAction::MenuRole::NoRole</enum>
    </property>
   </action>
   <action name="relation_create_action">
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>Связь</string>
    </property>
    <property name="menuRole">
     <enum>QAction::MenuRole::NoRole</enum>
    </property>
   </action>
   <action name="move_action">
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>Переместить</string>
    </property>
    <property name="menuRole">
     <enum>QAction::MenuRole::NoRole</enum>
    </property>
   </action>
   <action name="delete_action">
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>Удалить</string>
    </property>
    <property name="menuRole">
     <enum>QAction::MenuRole::NoRole</enum>
    </property>
   </action>
  </actiongroup>
 </widget>
 <resources/>
 <connections/>
</ui>
